### ISSUE TITLE
Elevation is not removed when I leave the region

### ISSUE THREAD

**Issue by Hazuma81** [caewok/fvtt-terrain-mapper#67]
Browser: I tested all brwoser (Chrome, Firefox , Opera GX , Edge)
Foundry vs: 12.331 stable release
Dnd Vs 3.3.1
Module Vs. 0.4.8

Entering a zone set as platue , it takes the elevation but exiting it remains at the same elevation, I attach video and pictures.

https://github.com/user-attachments/assets/4a05ecd1-84bb-4245-8364-d2c8730ef975

**Comment by Crusufix** [comment]
Not sure if my issue is related to this issue, but the symptom seems to be the same.

Browser: I tested Chrome, Firefox , and Edge
Foundry vs: 12.331 stable release
PF2e 6.4.1
Terrain Mapper 0.4.8

I've created a Difficult Terrain region, set the behaviour to Set Terrain: Difficult Terrain. The effect is added to the token upon entering the region, but it is not removed when leaving the region.

**Comment by ullantra** [comment]
#65 same issue as me

**Comment by jma3974** [comment]
Version 12 Build 331
PF2e 6.4.1
Terrain Mapper 0.4.8
Browser: Edge
 
+1 for this. I'm seeing this for creating Difficult and Very Difficult Terrain. The "condition" is applied. but doesn't come off when moving out of the space.

![image](https://github.com/user-attachments/assets/007ee62a-e762-4dfb-b6d9-5e05f7ddee5e)

**Comment by caewok** [comment]
I cannot replicate this. Do you have any other modules active other than TM and libWrapper?

**Comment by Hazuma81** [comment]
Yes, I have many but I tried using the find the culptist module and when I disable all the modules ,the problem remained. Now I am abroad on vacation and can't do a further test , I can give you confirmation on November 18 , when I come home

**Comment by Crusufix** [comment]
> I cannot replicate this. Do you have any other modules active other than TM and libWrapper?

I am able to replicate this in a brand new clean PF2e World with only TM and LibWrapper enabled.

![image](https://github.com/user-attachments/assets/3e7996ad-6f62-4454-9d5c-992e54364889)

**Comment by Commander-Fayne** [comment]
I have a similar issue. I have a plateau at 10 feet, and a ramp leading up to it. I walk the token into the plateau without the stairs, just going from 0 to 10 by climbing the wall. The plateau's only behavior is Terrain Mapper behavior set to plateau 10 feet highest elevation. When I drag the token to the plateau, it does not change elevation. When I drag it inside the plateau, the module resumes expected behavior and sets the elevation to 10, but when I drag it out of the plateau back to the ground, no amount of movement will set the elevation back to 0. 

If I drag it from the plateau to the stairs, however, the module takes over. Stairs seems to be working very well, it's just plateaus that has a performance bug. 

I'm also getting the -1000000 elevation, then scrolling back up to the assigned elevation bug when going from the top of stairs to the plateau again.

Which is really strange because this whole map was working perfectly last night. When I reloaded the world this morning is when I see all these issues. Maybe it has something to do with map reload?

**Comment by caewok** [comment]
Try v14.0.1.
